plugins/kubernetes-node/report.api.md

```diff
@@ -256,7 +255,7 @@ export class PinnipedHelper {
   constructor(logger: LoggerService);
   // (undocumented)
   tokenCredentialRequest(
-    clusterDetails: ClusterDetails_2,
```

Comment:
Oh my god this has been driving me wild why everything is always duplicated in the `api-reports`.

Turns out its partly this 🙃

Reply:
Indeed!

Reply:
It might make sense to make a lint rule against importing your own library. The tricky thing is, there may be some legitimate fringe uses of it across entrypoints etc.